vagrant/provisioning/roles/common/tasks/main.yml

```diff
@@ -26,6 +26,7 @@
       - tar
       - wget
       - sshpass
+      - firewalld
```

Comment:
The FIPS-enabled RHEL 7.5 VMs John created for EEOC did not have these firewall packages